flixel/graphics/frames/FlxBitmapFont.hx

```diff
@@ -79,6 +79,17 @@ class FlxBitmapFont extends FlxFramesCollection
 	 */
 	var frame:FlxFrame;
 
+	/**
+	 * Lookup tables for additional glyphs characters.
+	 */
+	public var lookupTable:Map<String, String> = [];
+	public var revLookupTable:Map<String, String> = [];
+
+	/**
+	 * Special characters start index.
+	 */
+	public var specialStart:Int = 500;
```

Comment:
I also think this can be implemented without needing a special start, and an ability to parse tokens to special characters seems helpful on it's own wether that maps to some large unicode index, or not